.evergreen.yml

```diff
@@ -41,6 +41,30 @@ functions:
           GOROOT: ${go_root}
           <<: *go_options
         command: make test
+  "atlas e2e":
+    - command: subprocess.exec
+        type: test
+        params:
+          add_to_path:
+            - ${go_bin}
+          working_dir: src/github.com/mongodb/mongocli/e2e/atlas
+          env:
+            WORKDIR: ${workdir}
+            GOROOT: ${go_root}
+            <<: *go_options
+          command: make e2e-test-atlas
+  "iam e2e":
```

Comment:
[q] any reason to split this here? you are increasing the challenges of the PR by just also changing how tests run which was working fine before